code/game/objects/items/storage/belt.dm

```diff
@@ -356,28 +342,25 @@
 		/obj/item/stack/sheet/bone,
 		/obj/item/lighter,
 		/obj/item/storage/box/fancy/cigarettes,
-		/obj/item/reagent_containers/food/drinks/bottle,
```

Comment:
Why is this removed

Reply:
They already have /obj/item/reagent_containers/food/drinks on the list, subtype pathing works for storages.